packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -377,6 +377,44 @@ import { Button, TextField } from 'ds-medicare-gov';
 
 </ThemeContent>
 
+<Alert heading="Why switch to Preact?">
+
+If you have a React project, chances are you can actually start using Preact today and save on your total bundle size. For our own component library, [we found that switching to Preact shrunk our bundle by about 92KB](https://github.com/CMSgov/design-system/discussions/2378). The Preact library itself is smaller than React, and so far its claim of full compatability with React has proven true.
+
+Check out [our preact-react-app example](https://github.com/CMSgov/design-system/tree/main/examples/preact-react-app) if you're curious how to make the switch in an existing React project without changing any of your source code.
+
+</Alert>
+
+If your project is based on Preact and not React, we've provided an import path for you to use the Preact version of our components directly. To import the Preact versions, add `/preact` to the end of your import path like this:
+
+<ThemeContent onlyThemes={['core']}>
+
+```js
+import { Button, TextField } from '@cmsgov/design-system/preact';
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+```js
+import { Button, TextField } from '@cmsgov/ds-healthcare-gov/preact';
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+```js
+import { Button, TextField } from 'ds-medicare-gov/preact';
+```
+
+</ThemeContent>
+
+### Using web components
+
+We're also starting to offer [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) as a framework-agnostic alternative to our React or Preact components. We will be working on expanding this library over time, but at a moment only a subset of our JavaScript components are represented in this library. The easiest way to consume these components is by [loading them from the CDN](https://github.com/CMSgov/design-system/tree/main/examples/cdn-web-components); however, they can also be installed via npm and imported by appending `/web-components` to your import path.
```

Comment:
I think this should be past tense as by the time someone is reading it this is already true. So `We've also begun (or started) offering`